cypress/integration/visitor_can_see_menu.feature.js

```diff
@@ -0,0 +1,19 @@
+describe("Visitor can see menu", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/products",
+      response: "fixture:products.json",
+    });
+  });
+
+  it("sucessfully", () => {
+    cy.visit("http://localhost:3001/");
+    cy.get("#product-1").within(() => {
```

Comment:
There should probably be assertions for more than one product, since it's supposed to show a list.